src/components/HistoryTable/HistoryTableContext.tsx

```diff
@@ -18,11 +22,31 @@ interface HistoryTableState {
   pageCount: number;
   currentPage: number;
   columnLimit: number;
+  historyTableFilters: TestFilter[];
+  setHistoryTableFilters: (filters: TestFilter[]) => void;
 }
 
 const HistoryTableDispatchContext = createContext<any | null>(null);
 
-const HistoryTableProvider: React.FC = ({ children }) => {
+interface HistoryTableProviderProps {
+  children: React.ReactNode;
+  initialState?: HistoryTableReducerState;
+}
+const HistoryTableProvider: React.FC<HistoryTableProviderProps> = ({
+  children,
+  initialState = {
```

Comment:
Extracted this out into a optional prop to make it easier to test the components that depend on this functionality. So we can pre seed the state.